ultravox/tools/ds_tool/ds_tool.py

```diff
@@ -23,6 +26,7 @@ class TtsTask:
     audio_column_name: Optional[str] = simple_parsing.field(default=None, alias="-a")
     voice: Optional[str] = simple_parsing.field(default=None, alias="-V")
     sample_rate: int = simple_parsing.field(default=16000, alias="-r")
+    format_fields: List[str] = simple_parsing.field(default_factory=list)
```

Comment:
not required anymore I believe.

Reply:
arghh, it was removed but somehow not saved properly before committing.